source/shared/net/guides/baker-pool.rst

```diff
@@ -0,0 +1,43 @@
+.. _baker-pool:
+
+================
+Baker management
+================
+
+A :ref:`baker <baker-concept>` is a node that participates in the network by baking (creating) new blocks that are added to the chain. There are some recommended best practices around baking, particularly when a baker runs a baker pool. A baker can choose to open a baker pool to which others can :ref:`delegate<delegation-concept>` some of their stake.
+
+Best practices for baking
+=========================
+
+As a baker you should manage your baker node responsibily for the benefit of all users of the blockchain. The recommendations for bakers are:
+
+- Make sure your node is running 24/7/365 in a reliable place. 
+- Protect your baker keys so that they are not lost or compromised.
+- Check regularly to make sure that you are not close to the bounding cap for max capital for a baker. If you're getting close to the max capital, you can split and make another baker to divide the capital.
+
```

Comment:
I think we should perhaps recommend having a pool open to delegation from others. This will only increase the return, as it increases the baker's effective stake, and the baker receives a commission on delegators' rewards.